apps/preview/next/pages/showcases/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,159 @@
+import { ShowcasePageLayout } from '../../showcases';
+// #region source
+import { SfButton, SfIconChevronLeft, SfIconChevronRight, usePagination } from '@storefront-ui/react';
+import classNames from 'classnames';
+
+export function Showcase() {
+  const { totalPages, pages, selectedPage, startPage, endPage, next, prev, onPageChange, maxVisiblePages } =
+    usePagination({
+      totalItems: 150,
+      currentPage: 2,
+      pageSize: 10,
+      maxPages: 1,
+    });
+
+  return (
+    <nav className="flex justify-between border-t border-neutral-200" role="navigation" aria-label="pagination">
+      <SfButton
+        type="button"
+        className="gap-3"
+        aria-label="Go to previous page"
+        disabled={selectedPage <= 1}
+        variant="tertiary"
+        slotPrefix={<SfIconChevronLeft />}
+        onClick={() => prev()}
+      >
+        <span className="hidden sm:inline-flex">Previous</span>
+      </SfButton>
+      <ul className="flex justify-center">
+        {!pages.find((page: number) => page === 1) && (
+          <li>
+            <div
+              className={classNames('flex pt-1 border-t-4 border-transparent', {
+                'font-medium border-t-4 !border-primary-500': selectedPage === 1,
+              })}
+            >
+              <button
+                type="button"
+                className="px-4 py-3 rounded-md text-neutral-500 hover:bg-primary-100 hover:text-primary-800 active:bg-primary-200 active:text-primary-900"
+                aria-current={selectedPage === 1}
+                onClick={() => onPageChange(1)}
+              >
+                1
+              </button>
+            </div>
+          </li>
+        )}
+        {startPage > 2 && (
+          <li>
+            <div className="flex border-t-4 border-transparent">
+              <button
+                type="button"
+                disabled
+                aria-hidden="true"
+                className="px-4 py-3 rounded-md text-neutral-500 hover:bg-primary-100 hover:text-primary-800 active:bg-primary-200 active:text-primary-900 "
+              >
+                ...
+              </button>
+            </div>
+          </li>
+        )}
+        {pages.map((page: number) => (
+          <>
```

Comment:
perhaps changing this to `li` tag and adding key here would resolve the problem. also you could use `<Fragment>` syntax and add key there

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2535/commits/3805a05d6d05b2db74b19ae6a9f4e6094aba1976